packages/ui/src/Layout/components/PluginItem.tsx

```diff
@@ -162,9 +193,31 @@ const PluginItem: React.FC<PluginItemProps> = ({
       classNames={classes}
     >
       <Accordion.Item value="photos" className={classes.pluginitem}>
-        <Accordion.Control h="35px" className={classes.control}>
-          <Text className={classes.buttonText}> {label}</Text>
-        </Accordion.Control>
+        <Flex
+          className={classes.collectionsHeader}
+          gap="5px"
+          justify="space-between"
+          align="center"
+          direction="row"
+          wrap="nowrap"
+          onMouseEnter={() => showActionButton(true)}
+          onMouseLeave={() => showActionButton(false)}
+        >
+          <Accordion.Control h="35px" className={classes.control}>
+            <Text className={classes.buttonText}> {label}</Text>
+          </Accordion.Control>
+          {isShowAction && (
+            <Flex
+              gap="sm"
+              justify="flex-end"
+              align="center"
+              direction="row"
+              wrap="wrap"
+            >
+              <ChannelAction />
+            </Flex>
+          )}
+        </Flex>
```

Comment:
looks like this can be handled with either a tooltip or a menu component from mantine

Reply:
No,, its not a menu or accordion.. its a conditional PLus icon that shows on hover then you click and see browse channel and create channel..